cmd/buckler/main.go

```diff
@@ -2103,9 +2126,9 @@ func main() {
 			)
 		*/
 		/* FIXME: support --long / -l and maybe --output / -o "fmt-str" */
-		tbl := tui.NewTable("UUID", "Name", "Summary", "Schedule", "Status", "Policy", "SHIELD Agent", "Target", "Store")
+		tbl := tui.NewTable("UUID", "Name", "Summary", "Schedule", "Status", "Policy", "SHIELD Agent", "Target", "Store", "Disaster Recovery")
 		for _, job := range jobs {
-			tbl.Row(job, job.UUID, job.Name, job.Summary, job.Schedule, job.Status(), job.Policy.Name, job.Agent, job.Target.Name, job.Store.Name)
+			tbl.Row(job, job.UUID, job.Name, job.Summary, job.Schedule, job.Status(), job.Policy.Name, job.Agent, job.Target.Name, job.Store.Name, job.DisasterRecovery)
```

Comment:
What do you think about putting a "Type" column before "Name", and setting it to "normal" if not a DR job, and "DR" if it is (probably with some coloring for boldness), vs. having a dedicated "Disaster Recovery" column?

Reply:
Type would make a bit more sense if we had more than two and if we expected operators to use fixed-key backups for more than just shield or a rare other case.